app/routes/not-found.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+export default Ember.Route.extend({
```

Comment:
Yeah the new syntax is as stickler says ::
```
import Route from '@ember/routing/route'

export default Route.extend({
```

Reply:
ah yeah my bad, let me fix all that!